agent/ebs/watcher.go

```diff
@@ -125,29 +214,6 @@ func (w *EBSWatcher) HandleEBSResourceAttachment(ebs *apiebs.ResourceAttachment)
 		})
 	}
 
```

Comment:
Should we be calling the daemonRunning() function here where this code has been deleted?

Reply:
No. `HandleEBSResourceAttachment` no longer managed the daemon task.